Tasks/KubernetesManifestV0/src/utils/FileHelper.ts

```diff
@@ -47,14 +47,28 @@ export function writeObjectsToFile(inputObjects: any[]): string[] {
                     tl.debug('Input object is not proper K8s resource object. Object: ' + inputObjectString);
                 }
             } catch (ex) {
-                tl.debug('Exception occurred while wrting object to file : ' + inputObject + ' . Exception: ' + ex);
+                tl.debug('Exception occurred while writing object to file : ' + inputObject + ' . Exception: ' + ex);
             }
         });
     }
 
     return newFilePaths;
 }
 
+export function writeJsonToFile(inputObjectString: string, kind: string, name: string): string {
```

Comment:
Rename this to `writeManifestToFile` and you can use this in the above function as well.